src/gettingstarted/template/cli-install.md

```diff
@@ -0,0 +1,58 @@
+# Start with a template
+
+## Next Steps: Installing the Platform.sh CLI
+
+With all of the requirements met, install the CLI to start developing with Platform.sh.
+
+<asciinema-player src="/videos/asciinema/verify-cli-extended.cast" preload=1 autoplay=1 loop=1></asciinema-player>
```

Comment:
Attributes should be quoted.  Even in HTML5, dagnabbit!

Reply:
In Firefox I am getting vertical and horizontal scrollbars on this, regardless of my screen size.  The output fits within the specified space, but I still get scrollbars.

Reply:
Once again, don't autoplay or loop.